driverdataaggregation/main.py

```diff
@@ -386,12 +411,29 @@ def aggregation_scaling_python(self,
                                     to request that the consumer application utilizes
                                     all physical cores for each client thread.
             :type cpu_hyperthreads: int, optional
+            :param languages: which language to use for the tester "cpp" or "fortran"
```

Comment:
Wording implies it's one or the other, but your code supports running with both

Reply:
changed to and/or